File: CCL-example-java/src/main/java/hohserg/ccl/example/CommonProxy.java
package hohserg.ccl.example;

import codechicken.lib.packet.PacketCustom;
import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;

public class CommonProxy {
    public void preinit(FMLPreInitializationEvent e) {
        //Даже, если у вас есть отдельный класс для серверного прокси, все равно регистрацию серверного обработчика делайте здесь
        PacketCustom.assignHandler(Main.modid/*имя канала не длиннее 20*/, new ServerPacketHandler());
    }
}

File: CCL-example-scala/src/main/scala/hohserg/ccl/example/CommonProxy.scala
package hohserg.ccl.example

import codechicken.lib.packet.PacketCustom
import net.minecraftforge.fml.common.event.FMLPreInitializationEvent

class CommonProxy {
  def preinit(e: FMLPreInitializationEvent): Unit = //Даже, если у вас есть отдельный класс для серверного прокси, все равно регистрацию серверного обработчика делайте здесь
    PacketCustom.assignHandler(Main.modid /*имя канала не длиннее 20*/ , new ServerPacketHandler)

}
